src/Repository/InvoiceRepository.php

```diff
@@ -10,5 +10,7 @@
 {
     public function get(string $invoiceId): InvoiceInterface;
 
+    public function findOneByOrderNumber(string $orderNumber): InvoiceInterface;
```

Comment:
There's a convention for repository methods:
 - `get*` - when it returns an entity or throws an exception
 - `find*` - when it returns an entity or null

The current implementation is more `get*`-like.